src/CMake/cpackLin.cmake

```diff
@@ -67,7 +67,7 @@ if (${LINUX_FLAVOR} MATCHES "^(Ubuntu|Debian)")
   else()
```

Comment:
Copyright date (line 2) needs to be updated.  Being that we are AMD, you might wish to add the following line:
`Copyright (C) 2022 Advanced Micro Devices, Inc.`